app/templates/components/filter-wrapper.hbs

```diff
@@ -1,4 +1,7 @@
-<div class="filter-header">
+<div
+  class="filter-header"
+  data-test-filter-section={{dasherizedFilterTitle}}
```

Comment:
@allthesignals What benefit is there to using a test selector here rather than just the actual class? (e.g. `.filter-section-community-district`)?

Reply:
@trbmcginnis good Q, the benefits are that these test-specific selectors are stripped out of the production build of the app. Using a regular class selector is fine too, until @andycochran needs to update or remove those classes entirely. Yeah, this would cause the test suite to fail and be caught, but it makes a CSS selector responsible for testing and styling all at once.

Reply:
Okay cool I'm going to go ahead and change those selectors in the test and re-commit, thanks!